examples/vae/utils/custom_mlp.py

```diff
@@ -56,24 +55,15 @@ def forward(self, *args, **kwargs):
         return [mm.forward(*args, **kwargs) for mm in self]
 
 
-def call_nn_op(op, epsilon):
+def call_nn_op(op):
     """
     a helper function that adds appropriate parameters when calling
     an nn module representing an operation like Softmax
 
     :param op: the nn.Module operation to instantiate
-    :param epsilon: a scaling parameter for certain custom modules
     :return: instantiation of the op module with appropriate parameters
     """
-    if op in [ClippedSoftmax]:
-        try:
-            return op(epsilon, dim=1)
-        except TypeError:
-            # Support older pytorch 0.2 release.
-            return op(epsilon)
-    elif op in [ClippedSigmoid]:
-        return op(epsilon)
-    elif op in [nn.Softmax, nn.LogSoftmax]:
+    if op in [nn.Softmax, nn.LogSoftmax]:
         return op(dim=1)
     else:
```

Comment:
nit: redundant `else`

Reply:
Let's refrain from pushing a fix tho. we can fix that after 0.2.0